src/app/.env.generated_staging

```diff
@@ -0,0 +1,8 @@
+IS_CONSOLE_LOGGING_ENABLED=false
```

Comment:
I would avoid create this file and find a way that generated_staging uses the staging file instead

Reply:
The env we pass should still be Staging, we prob just need to change the value in azure-pipeline.yml

Reply:
if you change the value in the azure-pipeline, it will cause conflicts when creating the artifacts

Reply:
How I see it when I helped @Lee31416, it's either we do:
- tweak the pipeline logic to set the environment & modify the pipeline logic to have a different artifact name
- modify the code to accept `generated_staging`

Reply:
Yeah, right, not a good thing

Reply:
Artifact name could be a parameter

Reply:
![image](https://github.com/nventive/FlutterApplicationTemplate/assets/57812285/05310205-72e6-4ef5-b007-de757a8673b1)

Reply:
generated_staging c'est pas un environment. Ça doit être staging. Le ArtifactName peut être différent de l'environment et pourrait donc être generated_staging

Reply:
Applied the change for the artifact name instead